### PR TITLE
redact some commands

### PR DESCRIPTION
**Pull request by maxtaco**

### CODE REVIEW

File: go/service/config.go [hunk 1]
```diff
@@ -292,7 +293,12 @@ func mergeIntoPath(g *libkb.GlobalContext, p2 string) error {
 }
 
 func (h ConfigHandler) HelloIAm(_ context.Context, arg keybase1.ClientDetails) error {
-	h.G().Log.Debug("HelloIAm: %d - %+v", h.connID, arg)
+	tmp := fmt.Sprintf("%v", arg.Argv)
+	re := regexp.MustCompile(`\b(chat|encrypt)\b`)
+	if mtch := re.FindString(tmp); len(mtch) > 0 {
+		arg.Argv = []string{arg.Argv[0], mtch, "(redacted)"}
```
Comment (patrickxb) [inline go/service/config.go]: this will make `keybase pgp encrypt -m "hi"` say `keybase encrypt (redacted)`

i don't really care, though...

Comment (maxtaco) [inline go/service/config.go]: yes, this was the exact intent!

Comment (patrickxb) [inline go/service/config.go]: it loses `pgp` is my point.

**Comment by zapu** [comment]
:+1:

**Comment by maxtaco** [comment]
fyi @cjb

**Pull request merged**